code/src/components/RangeSlider.js

```diff
@@ -0,0 +1,35 @@
+import React, {useState} from 'react'
+
+const Range = ({decideRange, IselectRange}) => {
+    const Satisfaction = () => {
+            if (decideRange <= 25) {
+                return ( <p className="satisfaction">Bad: {decideRange}</p>)
+            } else if (decideRange > 25 && decideRange <50) {
+                return ( <p className="satisfaction">Not so good: {decideRange}</p>)
+            } else if (decideRange > 50 && decideRange <75) {
+                return (<p className="satisfaction">Good: {decideRange}</p>)
+            } else if (decideRange > 75 ) 
+            { return (<p className="satisfaction">Very good: {decideRange}</p>)
+            }
+    }
```

Comment:
Love your function here! I tried to do the same but didn't manage to do it correctly. Great job! 💯